src/main/java/com/getindata/connectors/http/internal/table/lookup/JavaNetHttpPollingClient.java

```diff
@@ -74,15 +96,43 @@ public Optional<RowData> pull(RowData lookupRow) {
         }
     }
 
-    // TODO Add Retry Policy And configure TimeOut from properties
```

Comment:
I think the intent of this comment is to make use of the Flink Retry strategies, which includes async.

Reply:
The thing is that AsyncSink uses totally different, dedicated mechanism for retries => Flink native support. 
And actually there is separate TODO for this in HttpSinkWriter.

What kind of Flink retry strategies you were here @davidradl ?